charts/posthog/values.yaml

```diff
@@ -773,6 +773,52 @@ prometheus-kafka-exporter:
   kafkaServer:
     - posthog-posthog-kafka:9092
 
+
+###
+###
+### ---- prometheus-postgres-exporter ----
+###
+###
+prometheus-postgres-exporter:
+  # -- Whether to install the `prometheus-postgres-exporter` or not.
+  enabled: false
+
+  # -- Map of annotations to add to the pods.
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/path: /metrics
+    prometheus.io/port: "9187"
+
+  # -- Configuration options.
+  config:
+    datasource:
+      host: posthog-posthog-postgresql
+      user: postgres
```

Comment:
Do we offer setting this in the helm chart to something that isn't `postgres`?

Reply:
I'm not sure but I can remove it to simply default to https://github.com/helm/charts/blob/390ee6614b4bb0da96b397f9b382e3ddca9f59a4/stable/prometheus-postgres-exporter/values.yaml#L66

Reply:
I was keeping it to make clear here's a user/password block